notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<p>Nit: build in -&gt; built-in</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I kept this cell for no particular reason, but to be honest I find the way it shows the data super clumsy, inconsistent with other jobs and somewhat unrelated to the storyline, so I might remove it altogether.

Reply:
I removed it in the end.